code/modules/networks/computer3/bank_rec.dm

```diff
@@ -0,0 +1,1050 @@
+				src.print_text("What is the reason for this staff bonus? Enter '0' to re-select amount.")
+				return
+
+			var/bonus_total = length(src.bonus_crew) * src.bonus_amount
+			logTheThing(LOG_STATION, usr, "issued a bonus of [src.bonus_amount][CREDIT_SIGN] ([bonus_total][CREDIT_SIGN] total) to team [src.teams[src.bonus_team]].")
+			command_announcement(
+				"Bonus of [src.bonus_amount][CREDIT_SIGN] issued to all [src.teams[src.bonus_team]] staff.<br>Reason: [inputText]",
+				"Payroll Announcement by [src.authenticated] ([src.account.assignment])"
+			)
+			global.wagesystem.station_budget -= bonus_total
+			global.wagesystem.last_issued_bonus_time = world.time
+			for(var/datum/db_record/R as anything in src.bonus_crew)
+				// we used to tax the clown but that just put money from the budget into the aether vOv
+				R["current_money"] = (R["current_money"] + src.bonus_amount)
+			src.bonus_amount = 0
+			src.bonus_crew.len = 0
+			src.menu = MENU_STATION_BUDGET
+			src.print_budget()
+			return
+
+/datum/computer/file/terminal_program/bank_records/proc/mainmenu_text()
+	var/dat = {"<center>B A N K B O S S 2</center><br>
+	Welcome to BankBoss 2<br>
+	<b>Commands:</b>
+	<br>([MENU_MAIN_OPT_INDEX]) View bank records.
+	<br>([MENU_MAIN_OPT_SEARCH]) Search for a record.
+	<br>([MENU_MAIN_OPT_BUDGET]) View station budget.
+	<br>([MENU_MAIN_OPT_QUIT]) Quit."}
+
+	return dat
+
+/datum/computer/file/terminal_program/bank_records/proc/print_budget()
+	var/total_funds = global.wagesystem.station_budget + global.wagesystem.research_budget + global.wagesystem.shipping_budget
+	src.print_text("<br><b>Station Budget</b>")
+	src.print_text("Payroll Budget: [num2text(round(global.wagesystem.station_budget),50)][CREDIT_SIGN]")
+	src.print_text("Shipping Budget: [num2text(round(global.wagesystem.shipping_budget),50)][CREDIT_SIGN]")
+	src.print_text("Research Budget: [num2text(round(global.wagesystem.research_budget),50)][CREDIT_SIGN]")
+
+	var/payroll = 0
+	for(var/datum/db_record/R as anything in data_core.bank.records)
+		payroll += R["wage"]
+	var/surplus = round(global.wagesystem.payroll_stipend - payroll)
+	src.print_text("<br><b>Payroll Details</b>")
+	src.print_text("Payroll Stipend: [num2text(round(global.wagesystem.payroll_stipend), 50)][CREDIT_SIGN]")
+	src.print_text("Payroll Cost: [num2text(round(payroll),50)][CREDIT_SIGN]")
+	if(surplus >= 0)
+		src.print_text("<b>Payroll Surplus:</b> +[num2text(round(surplus),50)][CREDIT_SIGN]")
+	else
+		src.print_text("<b>Payroll Deficit:</b> -[num2text(round(surplus),50)][CREDIT_SIGN]")
+
+	var/profit_margin = total_funds - global.wagesystem.total_stipend
+	src.print_text("<br><b>Station Profitability</b>")
+	src.print_text("Total Budget: +[num2text(round(total_funds),50)][CREDIT_SIGN]")
+	src.print_text("Total Stipend: -[num2text(round(global.wagesystem.total_stipend), 50)][CREDIT_SIGN]")
+	src.print_text("<b>Profit Margin:</b> [num2text(round(profit_margin), 50)][CREDIT_SIGN]")
```

Comment:
What is this actually representing and how is it different to the payroll figures?

Reply:
These are the numbers used for the end-of-round budget score. The stipend here is the total amount across the entire shift, whereas the payroll stipend is per cycle. I'll modify the wording to help differentiate it.

Reply:
4c934a870934fdf55240a071754eaf2f76c50296